source/css/scss/_button-vars.scss

```diff
@@ -1,29 +1,31 @@
 @import 'palette';
 
 $button-function-factor: 0;
-$button-bg-color: $kiva-accent-blue;
+$button-bg-color: $blue;
 $button-bg-hover: $kiva-textlink-hover;
```

Comment:
elsewhere (e.g. on the register and login pages) I believe we are setting the bg hover color to dark blue - do I have that wrong?